sqltree/parser.py

```diff
@@ -188,6 +188,15 @@ class CharType(Node):
 CastType = Union[FunctionCall, CharType, Keyword, KeywordSequence]
 
 
+def _get_cast_type_keyword_number(casttype: CastType) -> int:
+    if isinstance(casttype, Keyword) or isinstance(casttype, FunctionCall):
```

Comment:
Wouldn't function calls be more than one token?

Reply:
True.. deleted the function